src/Home.js

```diff
@@ -15,7 +15,7 @@ class Home extends React.Component {
     super();
     this.state = {
       isLoggedIn: hoodie.account.isSignedIn(),
-      userModules: users.students[3].tracked_modules,
+      userModules: users.students[0].tracked_modules,
```

Comment:
This 0 index could come from a config, but at least all the occurences of this now point to the same json again